src/pages/ServiceDetails/ServiceInfo/__tests__/__snapshots__/ServiceInfoDescription.test.tsx.snap

```diff
@@ -23,15 +23,25 @@ exports[`#ServiceInfoDescription render correctly with data should render servic
           Service Overview
            
         </Title>
-        <Stack>
+        <Stack
+          gutter="md"
+          style={
+            Object {
```

Comment:
This `Object { ` seems a bit strange to me. Is this needed?

Reply:
Mmm the point is that this was updated by yarn test -u ...let me check why

Reply:
This object seems pretty normal to me, in the snapshot context. The style field needs an object to be defined. Though, it is not necessary in the Component itself. 
I've also gone through the other snapshots an I see the ones having the style field also has the object next to it.

Reply:
I don't think it is needed but since it is generated we won't worry about it.